### PR TITLE
Bump storybook to v6. Misc story fixes.

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

**Comment by ryan-ludwig** [comment]
In my testing things seemed to work pretty well. The biggest change is that Addon-knobs is being replaced by "controls", but knobs will continue to work until v7. We can wait until the docs are a little better fleshed out to make any chagnes. https://storybook.js.org/docs/vue/essentials/controls

File: .storybook/storybookStyles.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+	background: transparent;
```
Comment (ryan-ludwig) [inline .storybook/storybookStyles.scss:2]: This lets us test against background colors in storybook
image

**Comment by ryan-ludwig** [comment]
Ohh, seeing a few knob issues on loan card stories, www-page, and promos-generic-global-promo-banner

**Comment by eddieferrer** [comment]
@ryan-ludwig You can remove the computed styling variable in .storybook/stories/KvSettingsCard.stories.js the only reason i did that was to visualize the loan card with a dark background

File: .storybook/main.js [hunk 2]
```diff
@@ -3,12 +3,10 @@ module.exports = {
 		'./stories/**/*.stories.(js|mdx)'
```
Comment (eddieferrer) [inline .storybook/main.js:3]: This should now be: `'./stories/**/*.stories.@(js|mdx)'`
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#correct-globs-in-mainjs

File: .storybook/storybookStyles.scss (hunk 1 above)
Comment (mcstover) [reply .storybook/storybookStyles.scss:2]: rad!

**Pull request merged**